__tests__/Unit/pages/Standup/Standup.test.tsx

```diff
@@ -0,0 +1,92 @@
+import StandUp from '@/pages/standup';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import * as AuthenticatedHooks from '@/hooks/useAuthenticated';
+import {
+    userLoggedIn,
+    userNotLoggedIn,
+    notLoading,
+} from '../../../../__mocks__/db/standup';
+
+describe('Standup', () => {
+    test('should be able to render standup component if user is  logged-in and authenticated', () => {
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(
+            () => userLoggedIn
+        );
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('Loading...')).toBeInTheDocument();
+        expect(getByText('Standup')).toBeInTheDocument();
+    });
+
+    test('should not be able to render standup component if user is not logged-in and not authenticated', () => {
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(
+            () => userNotLoggedIn
+        );
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('You are not Authorized')).toBeInTheDocument();
+    });
+
+    test('should not be able to render standup component if user is logged-in and loading is false', () => {
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(
+            () => notLoading
+        );
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('Standup')).toBeInTheDocument();
```

Comment:
why is this test same as above test what is the difference here.

Reply:
No, the test is not same because here it's testing that if a user is logged in and loading is false then it should render the standup component.

Reply:
if you want I can remove this but removing this test will skip one line